server/controllers/tasks/lib/get_new_tasks.js

```diff
@@ -1,33 +1,90 @@
+import { intersection } from 'lodash-es'
 import _ from '#builders/utils'
 import { getEntitiesByClaim } from '#controllers/entities/lib/entities'
+import getEntitiesByUris from '#controllers/entities/lib/get_entities_by_uris'
 import getOccurrencesFromEntities from '#controllers/entities/lib/get_occurrences_from_entities'
-import getOccurrencesFromExternalSources from '#controllers/entities/lib/get_occurrences_from_external_sources'
+import { getOccurrencesFromExternalSources } from '#controllers/entities/lib/get_occurrences_from_external_sources'
 import { haveExactMatch } from '#controllers/entities/lib/labels_match'
+import { prefixifyInv } from '#controllers/entities/lib/prefix'
+import properties from '#controllers/entities/lib/properties/properties_values_constraints'
 import { getEntityNormalizedTerms } from '#controllers/entities/lib/terms_normalization'
 import typeSearch from '#controllers/search/lib/type_search'
-import { automerge } from './automerge.js'
+import { isNonEmptyArray } from '#lib/boolean_validations'
+import { forceArray } from '#lib/utils/base'
+import { automerge, validateAndAutomerge } from './automerge.js'
+import { findAuthorsWithIsbnsInWikipediaArticles } from './find_authors_with_isbns_in_wikipedia_articles.js'
 
 export default async function (entity, existingTasks) {
-  const [ newSuggestions, suspectWorksData ] = await Promise.all([
-    searchEntityDuplicatesSuggestions(entity, existingTasks),
+  const [ newSuggestionsSearchResults, suspectWorksData ] = await Promise.all([
+    searchEntityDuplicatesSuggestions(entity),
     getAuthorWorksData(entity._id),
   ])
-  if (newSuggestions.length <= 0) return []
+  if (newSuggestionsSearchResults.length <= 0) return []
   const { labels: worksLabels } = suspectWorksData
-  return Promise.all(newSuggestions.map(addOccurrencesToSuggestion(suspectWorksData)))
+  const suggestions = await getSuggestionsEntities(newSuggestionsSearchResults)
+
+  const suggestionWithIsbnInWpArticle = await findAuthorsWithIsbnsInWikipediaArticles(suspectWorksData, suggestions)
+  if (suggestionWithIsbnInWpArticle) {
+    return automerge(entity.uri, suggestionWithIsbnInWpArticle.uri)
+  }
```

Comment:
This looks like it belongs to `filterOrMergeSuggestions`, but that could be part of a later refactor, along with an async/await refactor of the `.then` chain hereafter

Reply:
This early merge is here to avoid getting occurrences from external services, while it is already known suspect can be merged